oggm/core/centerlines.py

```diff
@@ -144,12 +144,13 @@ def set_line(self, line):
     def flows_to_indice(self):
         """Indices instead of geometry"""
 
+        ind = []
         tofind = self.flows_to_point.coords[0]
         for i, p in enumerate(self.flows_to.line.coords):
             if p == tofind:
-                ind = i
-        assert ind is not None
-        return ind
+                ind.append(i)
+        assert len(ind) == 1
```

Comment:
```suggestion
        assert len(ind) == 1, 'We expect only one line to be found here'
```

I just learned about this [here](https://softwareengineering.stackexchange.com/questions/225956/python-assert-vs-if-return) and it just blows my mind I didn't know about this earlier.

Asserts are bad practice because their error message is obscure, but the possibility to actually add a message makes them much more useful in such cases!!!!